src/data/sensors.cpp

```diff
@@ -21,6 +21,10 @@ void Sensors::calibrate() {
     adxl2.calibrate();
 }
 
+double Vector3::norm() const {
```

Comment:
Why defining this in a different TU than its declaration ? Might be better to define in data.cpp, or even consider making it inline in data.h, as it is a helper function of Vector3.